src/ChainSafe.Gaming.InProcessTransactionExecutor/InProcessTransactionExecutor.cs

```diff
@@ -13,12 +13,22 @@
 
 namespace ChainSafe.Gaming.InProcessTransactionExecutor
 {
+    /// <summary>
+    /// Concrete Implementation of <see cref="ITransactionExecutor"/> using In Process.
```

Comment:
Same here. Pretty odd 'In Process' usage. Maybe re-phrase it some how.

Reply:
Yep, updated.